leaderelection/src/main/java/io/scalecube/leaderelection/RaftStateMachine.java

```diff
@@ -0,0 +1,58 @@
+package io.scalecube.leaderelection;
+
+
+import unquietcode.tools.esm.EnumStateMachine;
+
+/**
+ * RAFT state machine dealing with state transition FOLLOWER, CANNIDATE, LEADER.
+ * 
+ * @author Ronen Nachmias
+ */
+public abstract class RaftStateMachine {
```

Comment:
no - its just that RaftLeaderElection is a StateMachine so it extends StateMachine